Userland/Libraries/LibWeb/Page/EventHandler.cpp

```diff
@@ -890,7 +890,8 @@ bool EventHandler::handle_keydown(UIEvents::KeyCode key, u32 modifiers, u32 code
 
     if (auto* element = m_navigable->active_document()->focused_element(); is<HTML::HTMLMediaElement>(element)) {
         auto& media_element = static_cast<HTML::HTMLMediaElement&>(*element);
-        media_element.handle_keydown({}, key).release_value_but_fixme_should_propagate_errors();
+        if (!media_element.handle_keydown({}, key).release_value_but_fixme_should_propagate_errors())
+            return false;
```

Comment:
Returning false here tells the UI layer it should try to handle the key-down event. That probably results in no action, but it sounds like we want to return true here to say the event was handled by WebContent?

Also it took me a second to figure out what returning true/false from `HTMLMediaElement::handle_keydown` implied. I think it would make more sense to return true if the media element handled it, and false if it did not. 

(Though ultimately, I've wanted to turn these booleans into a clearer enum for a long time now - it's always confusing what returning true/false from these functions implies)

Reply:
I wrote this to match the behavior of `fire_keyboard_event()`, but I agree it is confusing, as the intent isn't really clear. 

I'll introduce an enum in both these places.

Reply:
> I'll introduce an enum in both these places

I've taken a look at this and I think it would be good to introduce an enum to all `handle_fooevent()` methods that currently return a `bool`. I think it would be better to do it in a separate PR, as this requires a fairly substantial code change.

> Ha not true actually. We don't ignore key events with modifiers here (a separate issue). So by returning false here, if I hit cmd+shift+m, we both mute the element and open the task manager

I've updated the PR so that media elements don't handle keypresses if any modifier is pressed.

Reply:
We still need to return true from here.